agent/functional_tests/util/utils.go

```diff
@@ -246,14 +247,20 @@ func (agent *TestAgent) startAWSVPCTask(taskDefinition string) (*TestTask, error
 		return nil, err
 	}
 
-	agent.t.Logf("Starting 'awsvpc' task in subnet: %s", subnet)
+	securityGroup, err := getSecurityGroupID()
```

Comment:
as per #1206 , you don't need this anymore, right?

Reply:
ah, for some reason I lost my mind, the metadata endpoint doesn't need to go through eni. I'll remove that.